admin-console/src/main/webapp/src/components/SystemBehaviorGroups/CreateEditBehaviorGroupModal.tsx

```diff
@@ -0,0 +1,85 @@
+
+interface CreateEditModalProps {
+    isEdit: boolean;
+    showModal: boolean;
+    isOpen: boolean;
+    initialSystemBehaviorGroup?: Partial<BehaviorGroup>;
+    isLoading: boolean;
+    onClose: () => void;
+    onSubmit: (systemBehaviorGroup: Partial<BehaviorGroup>) => void;
+}
+
+export const CreateEditBehaviorGroupModal: React.FunctionComponent<CreateEditModalProps> = (props) => {
+
+    const [ systemBehaviorGroup, setSystemBehaviorGroup ] = React.useState<Partial<BehaviorGroup>>(props.initialSystemBehaviorGroup ?? {});
+
+    const actionOption = [
+        <FormSelectOption key={ 0 } isPlaceholder label='Choose an action' />,
+        <FormSelectOption key={ systemBehaviorGroup?.id } label='Send an email to Users: All' value={ systemBehaviorGroup?.id } />,
+        <FormSelectOption key={ systemBehaviorGroup?.id } label='Send an email to Users: Admins' value={ systemBehaviorGroup?.id } />
+    ];
+
+    const handleChange = (value: string, event: React.FormEvent<HTMLInputElement> | React.FormEvent<HTMLTextAreaElement>
+        | React.FormEvent<HTMLSelectElement>) => {
+        const target = event.target as HTMLInputElement | HTMLSelectElement;
+        setSystemBehaviorGroup(prev => ({ ...prev, [target.name]: target.value }));
+    };
+
+    const onSubmitLocal = React.useCallback(() => {
+        props.onSubmit(systemBehaviorGroup);
+    }, [ systemBehaviorGroup, props ]);
+
+    React.useEffect(() => {
+        setSystemBehaviorGroup(props.initialSystemBehaviorGroup ?? {});
+    }, [ props.initialSystemBehaviorGroup ]);
+
+    return (
+        <React.Fragment>
+            <Modal
+                variant={ ModalVariant.medium }
+                title={ `${ props.isEdit ? 'Update' : 'Create'} your System Behavior Group` }
+                isOpen={ props.showModal }
+                onClose={ props.onClose }
+            ><Form isHorizontal>
+                    <FormGroup label='Group Name' fieldId='displayName' isRequired
+                        helperText={ props.isEdit ? <HelperText><HelperTextItem variant="warning" hasIcon>
+                                                    If this field is modified it may affect exisiting behavior.
+                        </HelperTextItem></HelperText> : 'Enter a name for your group' }>
+                        <TextInput
+                            type='text'
+                            value={ systemBehaviorGroup.displayName }
+                            onChange={ handleChange }
+                            id='displayName'
+                            name="displayName"
+                        /></FormGroup>
+                    <FormGroup label='Action' fieldId='actions' isRequired>
+                        <FormSelect
+                            id='actions'
+                            name='actions'
+                            value={ systemBehaviorGroup.actions }
+                            open={ props.isOpen }
+                            onChange={ handleChange }
```

Comment:
I guess this case should be handled differently. Depending on the selected value we should set the properties of the action (maybe in a separate object as is not part of the `create behavior group request`)
email-all = `{ ignore_preferences: false, only_admins: false }`
email-admins = `{ ignore_preferences: false, only_admins: true }`

Other combinations would be
email-all-ignore-preferences = `{ ignore_preferences: true, only_admins: false }`
email-admin-ignore-preferences = `{ ignore_preferences: true, only_admins: true }`